CoreScriptsRoot/Modules/Stats/BarGraph.lua

```diff
@@ -19,31 +19,49 @@ local StatsUtils = require(CoreGuiService.RobloxGui.Modules.Stats.StatsUtils)
 local BarGraphClass = {}
 BarGraphClass.__index = BarGraphClass
 
-function BarGraphClass.new(showAverage) 
+function BarGraphClass.new(showExtras) 
   local self = {}
   setmetatable(self, BarGraphClass)
 
-  self._frame = Instance.new("Frame")
-  self._frame.Name = "PS_BarGraph"
-  self._frame.BackgroundTransparency = 1.0
+  self._barFrame = Instance.new("Frame")
+  self._barFrame.Name = "PS_BarFrame"
+  self._barFrame.BackgroundTransparency = 1.0
   
-  self._showAverage = showAverage
+  self._lineFrame = Instance.new("Frame")
+  self._lineFrame.Name = "PS_LineFrame"
+  self._lineFrame.BackgroundTransparency = 1.0
+  
+  self._showExtras = showExtras
   
   
   self._values = {}
   self._average = 0
+  self._target = 0
```

Comment:
could you briefly explain what target accomplishes? Having some trouble following thru in the code... is it for showing an ideal performance standard?